pages/authLogin/authLogin.ejs

```diff
@@ -62,6 +62,29 @@
     border-color: <%- config.shibLinkColors.active.border %>;
     color: <%- config.shibLinkColors.active.text %>;
   }
```

Comment:
All of the things above were used to brand the institution login buttons. I assume we are doing away with this? If so, I think we can eliminate all the `config.shibLinkColors` options.

Reply:
We found that some folks are still using these in production, so we'll leave them alone for now.